pegasus/routes/dev_routes.rb

```diff
@@ -68,3 +68,22 @@
   end
   'success'
 end
+
+post '/api/dev/check-dtsn' do
```

Comment:
We could handle both staging and staging-next in one hook, since github posts details for PRs against all branches to check-dts already.

This is fine for now though.